apps/admin/src/app/(api)/api/v1/sensor_input/route.ts

```diff
@@ -18,20 +18,37 @@ export const POST = async (req: NextRequest) => {
         const binaryData = await parseReadableStream(body);
         const data = SensorDataRequest.fromBinary(binaryData);
 
-        if (data.value < 0) {
-            return new NextResponse(SensorDataResponse.toBinary({ status: 200 }), {
-                status: 200,
-                headers: {
-                    "Content-Type": "application/x-protobuf",
+        if (data.value <= 0) {
+            return new NextResponse(
+                SensorDataResponse.toBinary({ status: 400, statusMessage: "Value is equal to or less than zero" }),
+                {
+                    status: 400,
+                    headers: {
+                        "Content-Type": "application/x-protobuf",
+                    },
                 },
-            });
+            );
         }
 
         try {
             const sensorId = await getSensorIdFromSensorToken(data.accessToken);
             const needsSum = data.type === SensorType.ANALOG_ELECTRICITY;
 
-            await insertSensorData({ sensorId, value: data.value, sum: needsSum });
+            try {
+                await insertSensorData({ sensorId, value: data.value, sum: needsSum });
+            } catch (e) {
+                if ((e as unknown as Error).message === "value/too-high") {
```

Comment:
Ist nicht ganz clean code, einen hard codet String zu verwenden für die References. Optimalerweise baut wie es Daniel z. B. schon mal gemacht hat eine eigene Error Klasse (oder nennt man das hier type?) oder lagert den String at least in eine  Ressource aus.
Nur fürs Protokoll, ist mir egal ob wirs einhalten :D